core/utils/src/main/kotlin/edu/stanford/spezi/core/utils/foundation/builtin/RepositoryValue.kt

```diff
@@ -0,0 +1,17 @@
+package edu.stanford.spezi.core.utils.foundation.builtin
+
+import edu.stanford.spezi.core.utils.foundation.RepositoryAnchor
+import edu.stanford.spezi.core.utils.foundation.knowledgesource.KnowledgeSource
+
+interface AnyRepositoryValue {
```

Comment:
With my proposal, this type erased interface and the `RepositoryValue` below is not needed as we can achieve that internally (see `ValueRepository`) in my comment

Reply:
This was replaced with Map.Entry<KnowledgeSource<Anchor, *>, Any> which is essentially equivalent.